QUICKSTART.md

```diff
@@ -56,4 +56,38 @@ Now the application backend can call Warehouse API to obtain:
 1. The current app version
 1. The asset urls
 
+```js
+const Request = require('wrhs/src/utils/request');
```

Comment:
Can we extract this object into a separate module? Otherwise, you're asking people to install our full CLI as a dependency in their app just to consume a tiny part of it.

Reply:
I can do as fast follow.